testdata/serviceconfig.yml

```diff
@@ -21,7 +21,7 @@ rules:
           method: api.method | "unknown"
           response_code: response.http.code | "unknown"
       - descriptor_name:  request_latency
-        value: response.latency | "unknown"
+        value: response.latency | 0
```

Comment:
can we use `"NaN"` here instead (i've verified that value works in config)? I worry that `0` will be handled incorrectly along the way... and actually be sent to backends.

we need something to signal the manager (or whatever) to not pass this value along here (or something).

@ZackButcher thoughts?